src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Remove Https configuration
+REM Note that we always return successful code in clean up routine because clean up errors are not fatal
+
+netsh http delete sslcert ipport=0.0.0.0:8084
```

Comment:
The ports will be passed from CMD to CMD via test properties. We have an issue opened to change self host service ports. Making ports configurable will be considered when we solve the issue.